scrapers/JimWeathersArchives.yml

```diff
@@ -0,0 +1,54 @@
+name: "Jim Weathers Archives"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - jimweathersarchives.com
+    scraper: sceneScraper
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - jimweathersarchives.com
+    scraper: performerScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: 
+        selector: //div[@class='title_bar']/span/text()
+      URL: 
+        selector: //div[@class='rearrangeTab options_block active']/a[@class='option_button']/@href
+      Details:
+        selector: //span[@class='update_description']
+      Date:
+        selector: //div[@class='gallery_info']/div[@class='table']/div[@class='row']/div[@class='cell update_date']
+        postProcess:
+          - parseDate: 01/02/2006
+      Image:
+        selector: //meta[@name="twitter:image"]/@content
+        postProcess:
+          - replace:
+              - regex: ^(.+)\-.+
+                with: $1-4x.jpg
+      Studio:
+        Name:
+          fixed: BondageCafe

```

Comment:
Is  the studio actually `BondageCafe` ?

Reply:
Based on the person who requested the scraper, yes. 
https://discord.com/channels/559159668438728723/651418567475986432/1014170965019729981